scripts/README.md

```diff
@@ -4,10 +4,6 @@ This package contains useful scripts for managing a Skynet Webportal.
 
 ## Available Scripts
 
-**blocklist-skylink.sh**\
-The `blocklist-skylink.sh` script adds a skylink to the blocklist on all
-servers.
-
```

Comment:
I don't think we want to remove this. I know we can use blocker but looking at this from the perspective of someone unfamiliar with the stack and working under pressure, I'd rather give them more than one tool, so they have a higher chance of finding it quickly.

Reply:
reverted changes and removed only the airtable scripts